PsfLauncher/main.cpp

```diff
@@ -141,15 +141,15 @@ int launcher_main(PCWSTR args, int cmdShow) noexcept try
         {
             fullargs = args;
         }
-
-        LogString("Process Launch: ", fullargs.data());
+        LogString("Process Launch: ", exePath.c_str());
+        LogString("     Arguments: ", fullargs.data());
         LogString("Working Directory: ", currentDirectory.c_str());
 
         //THROW_IF_FAILED(StartProcess(exePath.c_str(), (L"\"" + exePath.filename().native() + L"\" " + exeArgString + L" " + args).data(), (packageRoot / dirStr).c_str(), cmdShow, INFINITE));
         HRESULT hr = StartProcess(exePath.c_str(), (L"\"" + exePath.filename().native() + L"\" " + exeArgString + L" " + args).data(), (packageRoot / dirStr).c_str(), cmdShow, INFINITE);
         if (hr != ERROR_SUCCESS)
         {
-            Log("Error return from launching process.");
+            Log("Error return from launching process 0x%x.", GetLastError());

```

Comment:
GetLastError returns a decimal, not a hex.  Please remove the 0x

Reply:
I am going to disagree with the comment.  You can insist if you want, but...

The function returns an binary value, which is displayable as a decimal or hex form.  Most computer professionals, the only ones to be looking at the debug console port, would prefer to see this in hex form as this is the most traditional format for error codes.  This makes a return code of 0x70003 be familiar as a file/registry not found even though the error is more specific.

Reply:
Interesting.  Thank you for your input.  

I prefer to see decimal.  I wonder if I'm the minority.

Either way, keep the change.  If people start asking for decimal it can be changed.